blocks/newslist/newslist.css

```diff
@@ -48,17 +48,17 @@
 .newslist .newslist-container .newslist-item .newslist-item-footer a {
   font-family: var(--graphik-semibold);
   text-transform: uppercase;
-  font-size: 16px;
+  font-size: var(--body-font-size-xxs);
```

Comment:
@raymond-pogi  this `-body-font-size-xxs` is 14px, but you're replacing 16px with this, any specific reason?

Reply:
done.